cypress/e2e/common/search-bar.cy.js

```diff
@@ -40,30 +38,51 @@ describe('Search Bar', () => {
 
   describe('Typeaheads', () => {
     it('has no typeahead functionality in All Data', () => {
-      cy.get(searchInput).clear();
-      cy.get(searchInput).type('bank', { delay: 200 });
-      cy.get('@typeahead.all').should('have.length', 0);
+      cy.intercept(typeAheadRequest, { body: [] }).as('typeahead');
+      cy.findByPlaceholderText('Enter your search term(s)').clear();
+      cy.findByPlaceholderText('Enter your search term(s)').type('bank', {
```

Comment:
can we chain these off of the clear call?

Reply:
Good catch. Done in 69ffde1